### PR TITLE
Feature/minsulee2/more log infos

### PR DESCRIPTION
**Pull request by minsulee2**
Added more logs for p2p communication

Updates:
- Add logging for new peer list
- Add logging for updating node info in a tracker side
- Add logging for recording all node info in a tracker side

### CODE REVIEW

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

File: tracker-server/tracker.js [hunk 1]
```diff
@@ -12,16 +12,29 @@ const { BlockchainConsts } = require('../common/constants');
...
+
 class Tracker {
   constructor() {
     this.blockchainNodes = {};
+    this.setIntervalForNodeInfoRecord();
+  }
+
+  setIntervalForNodeInfoRecord() {
+    setInterval(() => {
+      logger.info(`${JSON.stringify(this.blockchainNodes, null, 2)}`);
```
Comment (platfowner) [inline tracker-server/tracker.js:25]: minor) what about adding a todo to turn this log to debug level after debugging?

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks!

File: tracker-server/tracker.js (hunk 1 above)
Comment (minsulee2) [reply tracker-server/tracker.js:26]: Right! I left a comment for this!

**Comment by minsulee2** [comment]
Thanks for the review! merging now !

**Pull request merged**